case-lib/opt.sh

```diff
@@ -87,7 +87,6 @@ func_opt_parse_option()
             echo -e '    -h |  --help'
             echo -e "\tthis message"
             _func_case_dump_descption
-            exit 0
```

Comment:
I think you direct `exit 2` and add some comment will more good for this problem  
So you don't need to modify other place

Reply:
Done.